lib/logger/utils.ts

```diff
@@ -206,3 +206,41 @@ export function withSanitizer(streamConfig: bunyan.Stream): bunyan.Stream {
 
   throw new Error("Missing 'stream' or 'path' for bunyan stream");
 }
+
+export function isValidLogLevel(
+  logLevelToCheck: bunyan.LogLevel
+): logLevelToCheck is bunyan.LogLevel {
+  const allowedValues: bunyan.LogLevel[] = [
+    'trace',
+    'debug',
+    'info',
+    'warn',
+    'error',
+    'fatal',
+    bunyan.TRACE,
+    bunyan.DEBUG,
+    bunyan.INFO,
+    bunyan.WARN,
+    bunyan.ERROR,
+    bunyan.FATAL,
```

Comment:
Aren't these duplicated?

Reply:
I added these to check if the user has inserted a number instead of a string.
For example `bunyan.TRACE = 10`

Reply:
I think that process.env values are always strings

Reply:
Removed these checks. I also think the user wont enter numbers.